### PR TITLE
Ukrainian common endings

### PR DESCRIPTION
**Pull request by Mrqsyaka**
Added _list.json, _groups.json, and language.json of the common Ukrainian endings. Practising these endigns, it is possible hto tremendously increase a typing speed🙃

### CODE REVIEW

File: frontend/static/languages/ukrainian_common_endings.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
+{
+    "name": "ukrainain common endings"
```
Comment (rizwanmustafa) [inline frontend/static/languages/ukrainian_common_endings.json:2]: Please add a comma at the end of this line.

Comment (Miodec) [inline frontend/static/languages/ukrainian_common_endings.json:2]: Replace spaces with `_`

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/languages/ukrainian_common_endings.json (hunk 1 above)
Comment (Mrqsyaka) [reply frontend/static/languages/ukrainian_common_endings.json:2]: Thank you!

Comment (Mrqsyaka) [reply frontend/static/languages/ukrainian_common_endings.json]: Thanks! Done

Comment (Mrqsyaka) [reply frontend/static/languages/ukrainian_common_endings.json]: Done

File: frontend/static/languages/ukrainian_common_endings.json [hunk 2]
```diff
@@ -0,0 +1,125 @@
+{
+    "name": "ukrainain_common_endings",
+    "leftToRight": true,
```
Comment (rizwanmustafa) [inline frontend/static/languages/ukrainian_common_endings.json:3]: Also add ligatures fields

Comment (Mrqsyaka) [reply frontend/static/languages/ukrainian_common_endings.json]: I am not sure what "ligatures fields" are

**Pull request merged**